models/articles.js

```diff
@@ -55,3 +55,20 @@ exports.selectCommentsByArticleId = (id) => {
       return rows;
     });
 };
+
+exports.insertCommentByArticleId = (articleId, article) => {
+  return checkArticleExists(articleId)
+    .then(() => {
```

Comment:
How are you checking if the username exists?

Reply:
Hey! I'm not, psql handles this for me (it returns a 23503 error with detail: Key (author)=(coolcat) is not present in table "users") because this is a foreign key. so this is caught by my psqlErrorHandler